algebre-II-notes.tex

```diff
@@ -96,8 +96,8 @@ \subsection{Division euclidienne}
 	$\mathcal{P}$ est non vide et minoré par $0$. Donc il existe $d \in \mathcal{P}$ tel que $d$ est minimal.
 	On note $Q$ un polynôme de $I$ de degré $d$. Soit $P \in I$. On réalise la division euclidienne de $P$ par $Q$:
 	\[ \exists (U, R) \in K[X] \times K[X] \mid P = UQ + R \text{ et } \deg(R) < \deg(Q) \]
-	On a que $R = P - UQ \in I$ et donc $\deg(R) \in \mathcal{P}$. Donc $\deg(R) \geq d$ par définition de $d$.
-	Donc $\deg(R) = d$ et donc $R = 0$ car $d$ est minimal. Donc $P = UQ$ et donc $I = <Q>$.
+	On a que $R = P - UQ \in I$  car $I$ est un idéal, et donc $\deg(R) < d$. Par définition de $d$,
+	on a $R = 0$ car $d$ est minimal. Donc $P = UQ$ et donc $I = <Q>$.
```

Comment:
```suggestion
	On a que $R = P - UQ \in I$  car $I$ est un idéal. Comme $\deg(R) < d$, par définition de $d$
	on a que $R = 0$ car $d$ est minimal. Donc $P = UQ$ et donc $I = <Q>$.
```

Reply:
resolved

Reply:
You responded too quickly, I just changed the suggestion: https://github.com/Yag000/algebre-II-notes/pull/8#discussion_r1457718094